docs/rules/a11y-no-generic-link-text.md

```diff
@@ -1,4 +1,6 @@
-# No Generic Link Text
+# No Generic Link Text [Deprecated]
```

Comment:
📣 Any "best practice" or existing practice for deprecating a rule? I've just marked the rule as deprecated and removed it from the config.

Reply:
I think if `github/a11y-no-generic-link-text` has been completely removed form the code base then you could just delete this document altogether!

Reply:
Oh okay! I haven't removed the code yet. For this immediate release I will keep it marked as deprecated, then delete it entirely in follow-ups.